src/utils/styleTagsFromHtmlExtracter.js

```diff
@@ -0,0 +1,83 @@
+import htmlparser from "htmlparser2"
+
+/**
+ * Extract styles from html markup, returning the exctacted
+ * code and mappings.
+ *
```

Comment:
What should it return if the input `code` is not HTML? Or if there is no `<style>` tag in the HTML?